### DISCUSSION TITLE
Pod delegation based on annotations?

### DISCUSSION THREAD

**Discussion by headyj** [liqotech/liqo#1901] [Q&A]
Hello,

We are currently implementing Argo Workflows and we have a multi cluster setup and some of workflow steps needs to be executed on a specific cluster.
Admiralty offers the ability to delegate pod execution on a specific cluster based on annotations: https://admiralty.io/blog/2019/01/17/running-argo-workflows-across-multiple-kubernetes-clusters/#scenario-b-multicluster-etl

is there any possibility to do such things using Liqo? I would prefer using Liqo over Admiralty as it is more actively maintained, but I don't know if Liqo can handle such scenario

**Comment by aleoli** [comment]
Hi @headyj! Sorry for the very late response; I invite you to join our [Slack workspace](https://liqo-io.slack.com/join/shared_invite/zt-h20212gg-g24YvN6MKiD9bacFeqZttQ) if you didn't yet.

You can use node selectors to do that. Liqo will add unique labels on each virtual node, and you can also specify additional labels to select groups of clusters.
